### DISCUSSION TITLE
gosto

### DISCUSSION THREAD

**Discussion by sosotristek** [community/community#20123] [Other Feature Feedback, Questions, & Ideas] [answered]
Como posso evitar renders desnecessários de componentes funcionais no React?

**Comment by math-matos** [comment, marked as answer]
Você pode  usar o _React.memo_ para evitar renders desnecessários. Ele memoriza o resultado renderizado do componente e só o renderiza novamente se as props mudarem.

![image](https://github.com/community/community/assets/106177721/bc0abdce-0a22-48f5-9d4d-bdc2404f4104)
